web-marketplace/src/pages/BuyCredits/BuyCredits.tsx

```diff
@@ -36,9 +36,7 @@ export const BuyCredits = () => {
 
   useNavigateToSlug(slug, '/buy');
 
-  const [paymentOption, setPaymentOption] = useState<PaymentOptionsType>(
```

Comment:
Switched from `useState` to `useAtom` to avoid prop drilling and simplify `paymentOption` state management across nested components in the BuyCredits flow.